PyRoute/Galaxy.py

```diff
@@ -550,6 +655,12 @@ def process_owned_worlds(self):
                 worldstar.ownedBy = (owner, ownedBy[0:4])
 
     def is_well_formed(self):
```

Comment:
Add `is_well_formed()` as a do-nothing or throws exception method to `AreaItem` so we know it's always there and can be extended as needed.

Reply:
Added as a stub